fuel-kotlinx-serialization/src/appleMain/kotlin/fuel/serialization/ResponseExtensions.kt

```diff
@@ -6,8 +6,8 @@ import fuel.HttpResponse
 import kotlinx.serialization.DeserializationStrategy
 import kotlinx.serialization.json.Json
 
-public fun <T : Any> HttpResponse.toJson(
-    json: Json = Json { allowStructuredMapKeys = true },
+public actual fun <T : Any> HttpResponse.toJson(
+    json: Json,
```

Comment:
[q] actual method doesn't support the default parameter?

Reply:
It triggered the Kotlin IR flaws for expect/actual.

Reply:
Okay!